components/Challenges.js

```diff
@@ -2,16 +2,56 @@
 
 import styled from "styled-components";
 import NoCard from "./NoCard";
+import { useState } from "react";
 
 export default function ChallengeList({ challenges }) {
+  const [expandedChallenge, setExpandedChallenge] = useState(null);
+
+  function toggleExpand(id) {
+    if (id === expandedChallenge) {
+      setExpandedChallenge(null);
+    } else {
+      setExpandedChallenge(id);
+    }
+  }
+
   return (
     <CardContainer>
       {challenges.length === 0 ? (
         <NoCard />
       ) : (
         challenges.map((challenge) => (
-          <Card key={challenge.id}>
+          <Card
+            key={challenge.id}
+            challenge={challenge}
+            onClick={() => toggleExpand(challenge.id)}
+          >
+            <h2>{expandedChallenge === challenge.id ? "▼" : "▶︎"}</h2>
             <h2>{challenge.title}</h2>
+
+            {expandedChallenge === challenge.id && (
+              <>
+                <p>
+                  <h3>Level: {challenge.level} </h3>
+                </p>
+                <p>
+                  <h3>Info: </h3>
+                  {challenge.description}
+                </p>
```

Comment:
I think it's not a good practice to nest an `h3` in a `p` tag. If you need a wrapper around the heading, use a `div` instead.